Gemfile

```diff
@@ -85,7 +85,7 @@ gem "inifile",                        "~>3.0",     :require => false
 gem "logging",                        "~>1.8",     :require => false  # Ziya depends on this
 gem "net_app_manageability",          ">=0.1.0",   :require => false
 gem "net-ping",                       "~>1.7.4",   :require => false
-gem "net-ssh",                        "~>3.0.2",   :require => false
+gem "net-ssh",                        "~>3.2.0.rc2",   :require => false
```

Comment:
@chriskacerguis the difference between the two is this pr (https://github.com/net-ssh/net-ssh/pull/377) that enables us to specify the ssh-agent we are using, the older version relay on environment variables alone.